src/components/MenuBar/Menu/Menu.test.tsx

```diff
@@ -71,20 +76,11 @@ describe('the Menu component', () => {
         expect(getByText('Start Recording')).toBeTruthy();
       });
 
-      it('should open a dialog box when clicked', () => {
+      it('should correctly update recording rules and display the snackbar when the user clicks on the Start Recording button', () => {
         const { getByText } = render(<Menu />);
         fireEvent.click(getByText('More'));
 
         fireEvent.click(getByText('Start Recording'));
```

Comment:
just a question: what made you choose to use react testing library as opposed to enzyme ` .simulate('click');` like on line 133 below?